### PR TITLE
Simplify nuimo app

### PR DESCRIPTION
**Pull request by grunskis**
There was support for controlling multiple Home Assistant devices (aka entities) in Nuimo App. I removed this support to simplify code + it's not really needed at this point.

### CODE REVIEW

File: nuimo_app/senic/nuimo_app/components.py [hunk 1]
```diff
@@ -51,43 +51,33 @@ class PhilipsHue(Component):
...
-        self.is_light_on = all(x == "on" for x in self.state.values())
+        logger.debug("%s state: %s brightness: %s", self.entity_id, self.state, self.brightness)
 
     def button_press(self):
         """
         Toggle the state of all lights.
 
         """
-        if self.is_light_on:
+        if self.state == "on":
```
Comment (larsblumberg) [inline nuimo_app/senic/nuimo_app/components.py]: No need to change now, but I suggest that we [use quotation marks as described here](http://stackoverflow.com/a/56190/543875). TL;DR: Use single quotes for symbols. Use double quotes for strings that are supposed to be read by the user on the screen or by the programmer on the console.

File: nuimo_app/senic/nuimo_app/components.py [hunk 2]
```diff
@@ -107,21 +97,18 @@ def rotation(self, value):
         """
         delta = int(value / 1800 * self.MAX_BRIGHTNESS_VALUE)
 
-        for entity_id, brightness in self.brightness.items():
-            new_value = min([max([0, brightness + delta]), self.MAX_BRIGHTNESS_VALUE])
-            logger.debug("%s brightness current: %s delta: %s new: %s", entity_id,
-                         brightness, delta, new_value)
+        new_value = min([max([0, self.brightness + delta]), self.MAX_BRIGHTNESS_VALUE])
```
Comment (larsblumberg) [inline nuimo_app/senic/nuimo_app/components.py]: Better readable if min and max values stand next to each other:

    new_value = max(0, min(self.MAX_BRIGHTNESS_VALUE, self.brightness + delta))

or

    new_value = min(max(self.brightness + delta, 0), self.MAX_BRIGHTNESS_VALUE)

Is it necessary to mix `[]` and `()`?

File: nuimo_app/senic/nuimo_app/components.py [hunk 3]
```diff
@@ -107,21 +97,18 @@ def rotation(self, value):
...
 
-            self.brightness[entity_id] = new_value
-
-        max_value = max(self.brightness.values())
+        self.brightness = new_value
 
         # turn off bulbs if brightness has reached 0
         args = dict(transition=0)
-        if max_value > 1:
+        if new_value > 1:
```
Comment (larsblumberg) [inline nuimo_app/senic/nuimo_app/components.py]: If it's intentionally not `> 0` then I'd like to read a comment here why `> 1` is necessary

File: nuimo_app/senic/nuimo_app/nuimo_app.py [hunk 4]
```diff
@@ -152,26 +152,20 @@ def call_service_callback(entity_id, response):
             logger.debug("service_call response for %s:", entity_id)
             logger.debug(pformat(response))
 
-            status = response["success"]
-            action.entity_updated(entity_id, status)
+            if response["success"]:
+                matrix_config = action.led_matrix_config
+            else:
+                matrix_config = LEDMatrixConfig(icons.ERROR)
```
Comment (larsblumberg) [inline nuimo_app/senic/nuimo_app/nuimo_app.py]: Don't know if you like it:

    matrix_config = action.led_matrix_config if response["success"] else LEDMatrixConfig(icons.ERROR)

File: nuimo_app/senic/nuimo_app/components.py (hunk 2 above)
Comment (grunskis) [inline nuimo_app/senic/nuimo_app/components.py]: It's not really mixing, but making a list and choosing min/max from that. Just checked that in modern python versions don't require to have a list as a parameter to min/max. Will update...

File: nuimo_app/senic/nuimo_app/nuimo_app.py (hunk 4 above)
Comment (grunskis) [inline nuimo_app/senic/nuimo_app/nuimo_app.py]: For me it's hard to read long lines and not obvious that it's a condition.

**Comment by grunskis** [comment]
@larsblumberg addressed your feedback

**Pull request merged**